src/tstester/ledpixel.spec.ts

```diff
@@ -0,0 +1,161 @@
+import { LedPixelCmd, LedPixelReg } from "../jdom/constants"
+import { TestDriver } from "./base"
+import { ServiceTester } from "./servicewrapper"
+
+export class LedPixelTestRoutine {
```

Comment:
why do we separate these into a separate class? shouldn't those be tests?

Reply:
Yeah, I'm using these classes as an organizational tool, similar to the Mocha suite and test construct. I'm not using Mocha here because I don't think the Mocha regression flow makes sense for physical tests. I envision something more like the commissioner, which could automatically detect connected devices and launch tests, with some kind of human interface for hardware-in-the-loop and human-in-the-loop testing, instead of a command-line program that runs a bunch of tests. That being said, perhaps it would make sense to build it so that the syntax is consistent with Mocha as much as possible.